### ISSUE TITLE
ollama support using litellm

### ISSUE THREAD

**Issue by ck-amrahd** [ai-christianson/RA.Aid#20]
Hi,
Is it possible to support local LLMs using litellm and ollama? Thank you.

**Comment by ai-christianson** [comment]
Local LLMs are currently supported using the openai-compatible provider. This should work with litellm, but we don't have litellm or ollama built-in at this point.

**Comment by leonj1** [comment]
Does this change need to happen if it is working as-is today using the openai-compatible argument?

**Comment by sosacrazy126** [comment]
https://ollama.com/blog/tool-support

**Comment by ai-christianson** [comment]
> Does this change need to happen if it is working as-is today using the openai-compatible argument?

Right, that's kind of how I saw it initially --we should be 100% compatible with litellm as-is. I think the only thing we could potentially add is running our own litellm proxy internally, but I'm not sure how significant of a UX improvement that'd be.
